### ISSUE TITLE
Add UX tests about the Wikidata extension

### ISSUE THREAD

**Issue by fgiroud** [OpenRefine/OpenRefine#3430]
This is a placeholder ticket so far.
Community feedback is welcome to understand if, what and how to test this extension properly.

![Screen Shot 2020-12-16 at 14 46 39](https://user-images.githubusercontent.com/6267288/102356498-92387e00-3fad-11eb-96f9-d872e7bf4271.png)

### Proposed solution

To be determined

**Comment by fgiroud** [comment]
@riteshsoni123 that's a tough one, in case you're interested you can dig in

**Comment by riteshsoni123** [comment]
Yes surely I would like to give it a try , but a little bit explanation about this issue would be helpful.
